usr/share/rear/output/ISO/Linux-i386/250_populate_efibootimg.sh

```diff
@@ -63,6 +63,7 @@ title Relax-and-Recover (no Secure Boot)
 EOF
 else
     # create a grub.cfg
+    grub2_set_root=cd0
```

Comment:
@pcahyna 
please provide a comment that tells about the reason behind
why that `grub2_set_root=cd0` must be there,
cf. "Code must be easy to understand" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
Done via
https://github.com/rear/rear/pull/2453/commits/77d04392f71a055ff31d1e4e52c4554110d38d48